config/routes.rb

```diff
@@ -42,7 +42,13 @@
 
     authenticate :user, ->(user) { user.has_role?(:tech_admin) } do
       mount Blazer::Engine, at: "blazer"
-      mount Flipper::UI.app(Flipper, { rack_protection: {} }), at: "feature_flags"
+
+      flipper_ui = Flipper::UI.app(Flipper) do |builder|
+        builder.use Rack::Protection, origin_whitelist: ["https://dev.to"]
```

Comment:
**suggestion:** We should be able to leverage our `URL` helper module here, so we don't add anything new that later needs to be generalized again.

```suggestion
        builder.use Rack::Protection, origin_whitelist: URL.url
```

Reply:
This is a great suggestion!